src/main/java/com/egg/sp/entities/Supplier.java

```diff
@@ -20,8 +18,7 @@
 public class Supplier extends Users{
 
     @NotBlank
-    @Size(min=20, message= "Please enter a longer description")
-    @Size(max=255, message= "Please enter a shorter description")
+    @Size(min=20, max=255, message= "Please enter a longer description")
```

Comment:
Error descriptions in Spanish please, the proyect is for three Neigborhoods from Mendoza, Argentina